lib/services/app_test.go

```diff
@@ -346,3 +346,24 @@ func TestInsecureSkipVerify(t *testing.T) {
 		require.Equal(t, tt.expected, result)
 	}
 }
+
+func TestGetPublicAddrConfig(t *testing.T) {
```

Comment:
This is testing a one line function that is just a map lookup, so it's unlikely to ever fail and probably won't ever catch a legitimate bug either.

What if we tested this via `NewApplicationFromKubeService` (see lib/srv/discovery/discovery_test.go) instead?